import_logs.py

```diff
@@ -436,20 +473,27 @@ def get(self, key):
     r'\s+"(?P<referrer>.*?)"\s+"(?P<user_agent>.*?)"'
 )
 
+# We need to double escape the word-boundry to work correctly, because https://docs.python.org/2/reference/lexical_analysis.html#string-literals
```

Comment:
Guess it's the same for Python 3 ? Maybe we can update the link to point to Python 3 docs instead then...

Reply:
Actually I removed this bit for python 3, will remove the comment as well.